modoboa/core/tests/test_checks.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+
+from ..checks import settings_checks
+
+
+class CheckSessionCookieSecureTest(SimpleTestCase):
+    @override_settings(USE_TZ=False)
+    def test_use_tz_false(self):
+        """
+        If USE_TZ is off provide one warning.
+        """
```

Comment:
I think we can keep 1-line comments on only 1 line:
```python
""""If USE_TZ is off provide one warning."""
```

Reply:
fixed